lib/Importer/pytorch/CachingGraphRunner.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "CachingGraphRunner.h"
+
+#include "PyTorchModelLoader.h"
+
+#include "glow/Support/Support.h"
+
+GraphInfo::GraphInfo(const torch::jit::Node *node)
+    : subgraph(node->g(at::attr::Subgraph)) {}
+
+int32_t CachingGraphRunner::addGraph(const torch::jit::Node *node) {
+  // TODO: just use node* as key
+  GraphInfo graphInfo(node);
+  auto graphId = nextGraphId_++;
```

Comment:
Since glow is maintain a service here, which might receive continous cmd from front end for years in theory, think nextGraphId_++ is not very safe right? Agreeing using *node is much better.